workflows-scripts/data-export/export-marc-bib-records/dataExport_exportMARCBibRecords.jmx

```diff
@@ -168,11 +168,11 @@
         <stringProp name="ThreadGroup.on_sample_error">continue</stringProp>
         <elementProp name="ThreadGroup.main_controller" elementType="LoopController" guiclass="LoopControlPanel" testclass="LoopController" testname="Loop Controller" enabled="true">
           <boolProp name="LoopController.continue_forever">false</boolProp>
-          <stringProp name="LoopController.loops">1</stringProp>
+          <intProp name="LoopController.loops">-1</intProp>
```

Comment:
What's the significance of the negative sign?

Reply:
I forgot to check the `Infinite` checkbox in the script in the previous commit. The negative sign means `Infinite` checkbox is _checked_. It should always be -1 otherwise test will not run for the timed duration in the Grafana.